R/tribble_paste.R

```diff
@@ -5,7 +5,13 @@ globalVariables(".rs.readUiPref", "datapasta") #ignore this function in R CMD ch
 #' @return The parsed table text. Useful for testing.
 #' @export
 #'
-tribble_paste <- function(){
+tribble_paste <- function(x) {
+
+  # if data is provided, try to use that to generate the tribble_paste()
+  if(!missing(x)) {
+    clipr::write_clip(x)
+  }
+
```

Comment:
Cool. I just took your tweet example and ran with it. `clipboard_table <- x` was my first guess but I wasn't sure what form `x` was expected to be in further down the line. I'll have a go at fixing it up tonight, I was rushed to get even that much written.